### ISSUE TITLE
Remove `filter` stage

### ISSUE THREAD

**Issue by dkryvi** [AlexeyKorkoza/credit-site-server#101]
In the `forEach` block below you can write something like
```
if (isValidFile(file)) {
  const model = sequelize.import(path.join(__dirname, file));
  db[model.name] = model;
}

return ;
```

Define after `basename` const  
```
const isValidFile = file =>
  file.indexOf('.') !== 0) &&
  (file !== basename) && (file.slice(-3) === '.js'
```

and skip `filter` stage

https://github.com/AlexeyKorkoza/credit-site-server/blob/7964e82507306d4aec861eeddbcaaee6833c6270/src/models/index.js#L18

**Comment by AlexeyKorkoza** [comment]
@dkryvi , Please, check the PR #108

**Comment by AlexeyKorkoza** [comment]
It's merged, #108
